test/runBTagAnalyzer_cfg.py

```diff
@@ -433,10 +454,57 @@
 #    BlobStreamerName = cms.untracked.string('TBufferBlobStreamingService')
 #)
 #process.es_prefer_BTauMVAJetTagComputerRecord = cms.ESPrefer("PoolDBESSource","BTauMVAJetTagComputerRecord")
+
+if options.usePrivateJEC:
+    
+    from CondCore.DBCommon.CondDBSetup_cfi import *
+    import os
+    dbfile=''
+    if options.runOnData: dbfile=options.jecDBFile+'_DATA'
+    else: dbfile=options.jecDBFile+'_MC'
+    print "\nUsing private SQLite file", dbfile, "\n"
+    process.jec = cms.ESSource("PoolDBESSource",CondDBSetup,
+		    connect = cms.string( "sqlite_file:"+dbfile+'.db'),
```

Comment:
@BrieucF @imarches to my taste, nothing should be hardcoded, including absolute paths. The 'jecDBFile' can contain the pathname as a configurable.